src/www/app/config.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Outline Authors
+// Copyright 2021 The Outline Authors
```

Comment:
I think we are supposed to keep the original date. Revert?

Reply:
Done in a77bce1.